### PR TITLE
Adding initial CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by penrods**
Add a CONTRIBUTING guide.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+```
+
+That's it!  The currently active branch of your repo will be submitted.
+
+
+## Updating an existing skill
+
+A skill that is already in the Marketplace can easily be updated with:
+```
+mycroft-msk update .
```
Comment (forslund) [inline CONTRIBUTING.md]: Shouldn't we promote the usage of `submit` for the update as well?

**Review by penrods — COMMENTED** [review]
All sounds good to me.  This is _still_ your paddock too!  :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+easy as:
+```
+cd <local-path-to-your-skill>
+mycroft-msk submit .
+```
+or:
+```
+mycroft-msk submit <local-path-to-your-skill>
+```
+
```
Comment (ChristopherRogers1991) [inline CONTRIBUTING.md:42]: Might want to add something like:

```
Note: If you have two-factor authentication enabled for github, you will need to generate a [personal access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) and use the `-t` flag. E.g.

    mycroft-msk -t submit <local-path-to-your-skill>

```

Comment (penrods) [reply CONTRIBUTING.md:42]: Done!

**Comment by forslund** [comment]
Since there's not been any new comments on this one for the past five days, can we merge it as it stands now?

**Pull request merged**